src/Chapter2.hs

```diff
@@ -621,7 +628,10 @@ Write a function that takes elements of a list only in even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [Int] -> [Int]
+takeEven [] = []
+takeEven [x] = [x]
+takeEven (x:_:xs) = [x] ++ takeEven xs
```

Comment:
Nice pattern matching skills 👏🏼 

It is always better to use `:` instead of `++` where possible for efficiency 🙂 
Here it is easier without creating the list of one element and contamination, but with adding `x` to the list directly:

```suggestion
takeEven (x:_:xs) = x : takeEven xs
```